File: DesignPatternsExercises/java/ChainOfResponsibilityDesignPattern/Exercise/GeneralSupport.java
public class GeneralSupport extends SupportChain {
	@Override
	public void handleIssue(IssueType issueType) {
		if(issueType == IssueType.General) {
			System.out.println("General support team will contact you shortly");
		} else {
			__________(2)__________.handleIssue(issueType);
		}
	}
}

File: DesignPatternsExercises/ruby/chain_of_responsibility_design_pattern/exercise/general_support.rb
require_relative 'support_chain'

class GeneralSupport < SupportChain
  def handle_issue(issue_type)
    if issue_type == :general
      puts 'General support team will contact you shortly'
    else
      @next_support_link.__________(2)__________(issue_type)
    end
  end
end
